pipeline-runner/R/gem2s-2-load_cellranger_files.R

```diff
@@ -1,32 +1,42 @@
-#' Read input folder of 10x data
+#' Read user input files
```

Comment:
Can you change the name of this file and the name of the test file associated with it to match the code change, which is that this also reads non-cellranger files?